.github/workflows/lint.yml

```diff
@@ -18,7 +18,7 @@ jobs:
             node-version: 12.13.0
       - name: Lint frontend code with ESLint
         run: |
-          pip install pipenv
+          curl -sSL https://install.python-poetry.org | python3 -
```

Comment:
Can this not be `pip` installed? (Fine if it can't!)

Reply:
It can be, but the official documentation strongly recommends not doing so https://python-poetry.org/docs/master/#installing-with-pip